general/server/media/shows.md

```diff
@@ -5,19 +5,19 @@ title: Shows
 
 # Shows
 
-The most common naming scheme for shows is categorizing the files by series and then season. Another common method is simply using series folders, especially for shows that are organized by air date and those without seasons. Adding the year at the end in parentheses will yield the best results when scraping metadata.
+The most common naming scheme for shows is categorizing the files by series and then season. Another common method is simply using series folders, especially for shows that are organized by air date and those without seasons. Adding the year at the end in parentheses along with a media DB ID will yield the best results when scraping metadata. The two DBs supported by Jellyfin are [IMDB](https://www.imdb.com/) and [TMBD](https://www.themoviedb.org/). Media management software such as [Sonarr](https://sonarr.tv/) can be configured to add these IDs in to the folder names. The examples below show the ID syntax `[idtype-ID]`.
```

Comment:
Avoid "jargon" and needless contractions/abbreviations like "ID" and "DB". It makes it more difficult for the average user to read and understand the documentation properly.

This also feels more like something that should live in a `tip` block, instead of in the main description.

Something like:

```markdown
> [!TIP]
> In order to help with identifying series, Jellyfin can make use of a media provider identifier.
> This can be specified in your show's folder name, for example as: `Series (2010) [tt0106145]`
```

Also, we avoid any references to software used for piracy, such as Sonarr. If you really want to add a note to replace that reference, something like "Consult your media manager of choice's documentation for ways to include this identifier automatically", but I don't even think that's really necessary, as it's outside the scope of use of Jellyfin itself.

Reply:
That's much better, replaced what I wrote with your suggestion.